### DISCUSSION TITLE
access control multiple keys contain

### DISCUSSION THREAD

**Discussion by ElliotHYLee** [directus/directus#21187] [Feature Requests]
### Summary

currenlty, access control can check collection.some_key "is one of " $CURRENT_USER.some_key. 

For instance, 
user.purchased_keys = ["a", "b", "c"]
and 
digital_item.key = "a"

Then, the access can be configured if digital_item.key "is one of " $CURRENT_USER.purchased_keys

Now, I wonder if this can be expanded to multiple keys. Like

some_bundled_digital_item.key = ["c", "d", "e"] 
then,
some_bundled_digital_item.key "has overlap" $CURRENT_USER.purchased_keys

### Basic Example

some_bundled_digital_item.key "has overlap" $CURRENT_USER.purchased_keys

### Motivation

a digital product can be reused and bundled with differnte package while i can reuse the digital product. I need to simplify the access control

### Detailed Design

?

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

..

### Alternatives

..

### Adoption Strategy

..

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
